src/styles/features/onboarding/onboarding.css

```
@@ -0,0 +1,510 @@
+
+	&:checked + .onboarding-toggle-slider {
+		background-color: var(--color-primary);
+	}
+
+	&:checked + .onboarding-toggle-slider::before {
+		transform: translateX(16px);
+	}
+
+	&:focus + .onboarding-toggle-slider {
+		@apply ring-2;
+		ring-color: var(--color-primary);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)